docs/static-config-helpers/get-md-files.js

```diff
@@ -85,6 +86,19 @@ function slugWithLink() {
   return slugTransformer;
 }
 
+function appendImageBasePath(ast) {
+  function visitor(node) {
+    if (!!node.value.match(/<img /) && stage !== 'development') {
+      node.value = node.value.replace(`src="`, `src="/${landerBasePath}`);
```

Comment:
Ah interesting. If I'm reading this correctly, we're basically transforming the AST to prepend images in stage and prod with `/open-source/renature`? Makes sense if so!